debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -431,18 +431,36 @@ private String asHstoreString(byte[] data) {
 
     private Object convertHstoreToJsonString(Column column, Field fieldDefn, Object data){
         return convertValue(column, fieldDefn, data, "{}", (r) -> {
+          logger.debug("in ANON: value from data object: ***" + data.toString() + "***");
+          logger.debug("in ANON: object type is: ***" + data.getClass() + "***");
             if (data instanceof String) {
+                logger.debug("in String....");
```

Comment:
Could you please make this and subsequent messages a little bit more verrbose?

Reply:
I have done some work to be more verbose, but if it's not enough, definitely let me know. For these 3 particular logger statements, I went ahead and removed them, as they felt redundant.